front/app/components/UI/FormComponents/index.tsx

```diff
@@ -115,17 +116,23 @@ export const Spacer = styled.div`
   height: 8px;
 `;
 
-const OptionalText = styled.span`
-  color: ${({ theme }) => theme.colors.tenantText};
+const OptionalText = styled.div`
+  color: ${({ theme }) => theme.colors.textSecondary};
   font-weight: 400;
```

Comment:
Should the optional label only display under the field name if the field name is long? Or is always having it displayed underneath on mobile intended? I'm wondering if there should be a bit more space between the title and the optional label as well?
![image](https://user-images.githubusercontent.com/33987955/204803605-28d2e294-b6b7-4b6e-a275-f12bb0ba2678.png)

Reply:
It wraps on mobile. Just added a little margin top.